delta/plugins/elasticsearch/src/test/resources/routes/elasticsearch-view-list-response.json

```diff
@@ -2,7 +2,7 @@
   "@context" : [
     "https://bluebrain.github.io/nexus/contexts/metadata.json",
     "https://bluebrain.github.io/nexus/contexts/search.json",
-    "https://bluebrain.github.io/nexus/contexts/all-metadata.json"
+    "https://bluebrain.github.io/nexus/contexts/listings-metadata.json"
```

Comment:
can we call this `search-metadata.json` instead?

Reply:
Is not a search, is for listings. Search is when using the endpoint `/v1/views/{org}/{proj}/{id}/_search`

Reply:
we already have the search context that is used for listings. I think it makes sense.